src/sandstorm/run-bundle.c++

```diff
@@ -1162,12 +1162,23 @@ private:
 
     // The environment inherited from the host is probably no good for us. E.g. an oddball
     // locale setting can crash Mongo because we don't have the appropriate locale files available.
+    //
+    // That said, there are a few environment variables that we do re-export.
+    char * http_proxy, * https_proxy;
```

Comment:
Generally in C++ it's recommended to associate the `*` with the type, i.e. `char* x` not `char *x`. This of course leads to confusion when declaring multiple variables at once as you are doing here, and generally the answer to that is: don't do that. :) That is, declare them on separate lines.